model/Image.php

```diff
@@ -233,10 +247,10 @@ public function SetRatioSize($width, $height) {
 		$heightRatio = $height / $this->height;
 		if( $widthRatio < $heightRatio ) {
 			// Target is higher aspect ratio than image, so check width
-			if($this->isWidth($width) && !Config::inst()->get('Image', 'force_resample')) return $this;
+			if($this->isWidth($width)) return $this;
```

Comment:
As long as `$Image->Link()` etc. still works it shouldn't matter should it? Surely no one is hard coding links to `/assets/Uploads/_resampled/ResizedImageWzEyMDAsODAwXQ==-Sample-IMG-Apples.jpg"` or the like? And if they were it looks like 3.2 includes a breaking change there anyway - the arguments in the generated file name are now... weird. Encoded some how?

Plus I thought 3.2 was not a proper SemVer release, but the point from which we start following SemVer going forward? It already contains lots of breaking changes?